apps/api/src/domain/models/user-management/contributor/commands/create-contributor/contributor-created.event.ts

```diff
@@ -0,0 +1,5 @@
+import { BaseEvent } from '../../../../shared/events/base-event.entity';
+
+export class ContributorCreated extends BaseEvent {
```

Comment:
You need
- export type ContributorCreatedPayload = CreateContributor;
- @CoscradEvent(...)

Reply:
done